### PR TITLE
Documentation with yard

### LINKED ISSUE #12: Add documentation with YARD
Currently, `winevt_c` is poor documentation.
We should add documentation for winevt_c classes and methods.

### PR DESCRIPTION
**Pull request by cosmo0920**
Fixes #12.

### CODE REVIEW

File: ext/winevt/winevt_subscribe.c [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+ *    "Application", "*[System[(Level <= 4) and TimeCreated[timediff(@SystemTime) <= 86400000]]]"
+ *  )
+ *  while true do
+ *    @subscribe.each do |eventlog, message, string_inserts|
+ *      puts ({eventlog: eventlog, data: message})
+ *    end
+ *    sleep(0.1)
+ *  end
+ *
+ * @see: https://docs.microsoft.com/en-us/windows/win32/api/winevt/nf-winevt-evtsubscribe
```
Comment (ashie) [inline ext/winevt/winevt_subscribe.c]: The trailing `:` for `@see` seems break the output document.

Comment (cosmo0920) [reply ext/winevt/winevt_subscribe.c]: Fixed in https://github.com/fluent-plugins-nursery/winevt_c/pull/13/commits/d1480632018c5e684e8e6dcc53faa855ec810ba7.

File: ext/winevt/winevt_bookmark.c [hunk 2]
```diff
@@ -68,6 +93,12 @@ rb_winevt_bookmark_initialize(int argc, VALUE* argv, VALUE self)
   return Qnil;
 }
 
+/*
+ * This function updates bookmark and returns Bookmark instance.
```
Comment (ashie) [inline ext/winevt/winevt_bookmark.c]: nitpick: The term `function` seems inappropriate for Ruby, use `method` instead.

Comment (cosmo0920) [reply ext/winevt/winevt_bookmark.c]: Oh!

**Pull request merged**